mycore-base/src/main/resources/xsl/functions/classification.xsl

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
+<xsl:stylesheet version="3.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+  xmlns:xlink="http://www.w3.org/1999/xlink" 
+  xmlns:fn="http://www.w3.org/2005/xpath-functions"
+  xmlns:xs="http://www.w3.org/2001/XMLSchema"
+  xmlns:mcrclass="http://www.mycore.de/xslt/classification"
+  exclude-result-prefixes="fn">
+  
+  <xsl:function name="mcrclass:current-label" as="element()?">
+    <xsl:param name="class" as="element()" />
+    <xsl:choose>
+      <xsl:when test="$class/label[@xml:lang=$CurrentLang]">
+        <xsl:sequence select="$class/label[@xml:lang=$CurrentLang]" />
+      </xsl:when>
+      <xsl:when test="$class/label[@xml:lang=$DefaultLang]">
+        <xsl:sequence select="$class/label[@xml:lang=$DefaultLang]" />
+      </xsl:when>
```

Comment:
should be the `<xsl:otherwise>` case already

Reply:
I decided for another default and return the first available label now